### ISSUE TITLE
Bug: [ERROR] Unable to load config file: error="read /config/config.json: is a directory"

### ISSUE THREAD

**Issue by erltho** [hashicorp/terraform-provider-hashicups#39]
### Terraform Version
terraform v1.0.5

### Affected Resource(s)
docker-compose.yml

### Terraform Configuration Files / Docker compose file
```yml
version: '3.7'
services:
  api:
    image: "hashicorpdemoapp/product-api:v4280cf7"
    ports:
      - "19090:9090"
    volumes:
      - ${PWD}/conf.json:/config/config.json
    environment:
      CONFIG_FILE: '/config/config.json'
    depends_on:
      - db
  db:
    image: "hashicorpdemoapp/product-api-db:v4280cf7"
    ports:
      - "15432:5432"
    environment:
      POSTGRES_DB: 'products'
      POSTGRES_USER: 'postgres'
      POSTGRES_PASSWORD: 'password'
```

### Debug Output

```
api_1  | 2021-09-02T18:56:57.001Z [ERROR] Unable to load config file: error="read /config/config.json: is a directory"
```

### Panic Output
```
docker_compose_api_1 exited with code 1
```

### Expected Behavior
docker-compose up should be successfully spinning up the container. 

### Actual Behavior
docker compose exited with code 1, Hashicups api is not reachable. Cannot continue the [toturial](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers)

### Steps to Reproduce
Follow the guide listed in toturial ["Setup and implement Read"](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers)

### Suggested solution

Change from "${PWD}/conf.json:/config/config.json" to "./conf.json:/config/config.json" in line [L#8](https://github.com/hashicorp/terraform-provider-hashicups/blob/eb3b833280d3eaa911ea99c9539c0c4b7ad6de37/docker_compose/docker-compose.yml#L8)

**Comment by im2nguyen** [comment]
Hi @erltho, thank you for reporting! Do you want to submit a PR to address this? Your suggested solution seems solid.

If not, I'll create one early next week 😄

**Comment by erltho** [comment]
Sure thing, I will fix it straight away :)

**Comment by im2nguyen** [comment]
Thanks for submitting the PRs. We need to do the same for the rest of the other branches referenced in the tutorial, but this is a good step forward 😄

**Comment by erltho** [comment]
@im2nguyen after looking more into it (and spinning up some new VMs because of some trouble with the docker-compose) I now find my VM with defined PWD out of the box. Then this issue is no longer relevant. I am not sure why I didnt have PWD defined the first time, when I was triggered to write this issue. 

WDYT? Should we continue the change or revert?

**Comment by im2nguyen** [comment]
Hey @erltho, I think we should keep it as `.`. It's a relative path and works for all systems. This was causing issues in our other repos too

**Comment by erltho** [comment]
Hi @im2nguyen, I think I was able to cover the repos mentioned throughout the tutorial. Could we close this issue now?

**Comment by im2nguyen** [comment]
Hey @erltho, yep! Thanks for creating the PRs for each one of the branches and for running through the tutorials!

**Comment by AniketKariya** [comment]
I am still facing the same error with the updated docker-compose.yml file. 

```
api_1  | 2022-03-20T13:49:45.544Z [ERROR] Unable to load config file: error="read /config/config.json: is a directory"
db_1   |
db_1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
db_1   |
db_1   | 2022-03-20 13:49:40.095 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1   | 2022-03-20 13:49:40.097 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db_1   | 2022-03-20 13:49:40.148 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1   | 2022-03-20 13:49:40.233 UTC [27] LOG:  database system was shut down at 2022-03-20 13:49:19 UTC
db_1   | 2022-03-20 13:49:40.251 UTC [1] LOG:  database system is ready to accept connections
docker_compose_api_1 exited with code 1
```

**Comment by ajithkumar-natarajan** [comment]
I had the same issue as OP. My initial guess was that something was wrong with copying the `conf.json`. I checked and it all seemed good. I then Googled and came across this issue and this fix was already there in my case.

**The solution to my issue was that I had to run the docker commands with `sudo` permission**. 
Docker was creating the volume for the container at `/var/lib/docker/volumes/<volume_name>` (I am running a Linux OS) and it did not have enough permission to create the directory and copy the conf.json to it. The error though was a bit misleading - _[ERROR] Unable to load config file: error="read /config/config.json: is a directory"_, same as the OP.
